docs/embedded/tutorials/using-webhooks.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Using Webhooks
+description: Use webhooks with SPE
+ms.date: 02/26/2024
+ms.localizationpriority: high
+---
+
+# Using Webhooks
+
+## Set Up Webhooks with SharePoint Embedded
+Webhooks are automated messages that are transmitted by an application when a trigger is activated. They can be used in SPE to enable the automation of workflows, the integration of systems, and to respond to events in real time.
+You will use webhooks to invoke the Azure Cognitive Service APIs from the application whenever an existing file is updated, or a new file is uploaded.
+To set up webhooks with your [current SharePoint Embedded application](/training/modules/sharepoint-embedded-create-app/), you need to
+1.	Create and register a webhook endpoint to get notifications whenever there is change in your container. This will be done using REST APIs.
```

Comment:
Don't explicitly number bullets. All bullets should be `1.` and let the rendering engine create the correct numbers.